guides/introduction_to_blocks.md

```diff
@@ -103,6 +104,35 @@ $code_blocks_essay
 
 $demo_blocks_essay
 
+You can also use the `gradio.update` function as a short-hand for using the component class's update method.
+
+Here is the previous demo rewritten with `gradio.update`:
+
+```python
```

Comment:
Rather than creating a whole new demo (that would be redundant with `blocks_essay`) I decided to just include the source code for this demo here

Reply:
That makes sense but I really think we should include this and any other demos in the docstring so that they're rendered in Try Examples. I think would make it much easier for users to understand

Reply:
You're right. I just added a new demo with this code snippet and put it in the docstring!